docs/input-manifest/external-templates.md

```diff
@@ -0,0 +1,82 @@
+Claudie allows to plug in your own templates for spawning the infrastructure. Specifying which templates are to be used is done at the provider level in the Input Manifest, for example: 
+
+```yaml
+apiVersion: claudie.io/v1beta1
+kind: InputManifest
+metadata:
+  name: genesis-example
+  labels:
+    app.kubernetes.io/part-of: claudie
+spec:
+  providers:
+    - name: genesiscloud
+      providerType: genesiscloud
+      templates:
+        repository: "https://github.com/berops/claudie-config"
+        tag: "v0.9.0" # optional
+        path: "templates/terraformer/genesiscloud"
+      secretRef:
+        name: genesiscloud-secret
+        namespace: secrets
+...
+```
+
+- if no templates are specified it will always default to the latest commit on the Master/Main branch of the respective cloudprovider on the berops repository (i.e. ` https://github.com/Despire/claudie-config`).
+
+- if templates are specified, but no tag is present it will default to the latest commit of the Master/Main branch of the respective repository.
```

Comment:
We should also mention, what's the default `path` in case no definition is provided in the `templates` stanza.

Reply:
Do you mean this? Or literally write the default "template" structure that will be inserted

https://github.com/berops/claudie/blob/cb419b8e39ab50c769167c8f6a9aa7fbc92cbc4b/docs/input-manifest/external-templates.md?plain=1#L24

Reply:
Apologies for not being clear enough, allow me to rephrase:
We should also mention, what's the default `path` in case no definition is provided in the `templates.path` stanza. Or is `templates.path` mandatory for `templates`?

Reply:
it is mandatory.

`templates.repository` and `templates.path` are mandatory, i.e this identify where templates for a provider can be found.

`templates.tag` is optional.

Reply:
Understood. All good then, thanks for the clarification.